client/js/Utils/sc_helper.js

```diff
@@ -149,18 +149,18 @@ ScHelper.prototype.getResult = function (action_addr) {
       clearTimeout(timer);
       resolve(null);
     }, 10_000);
-    template.tripleWithRelation(
+    template.quintuple(
       new sc.ScAddr(parseInt(action_addr)),
-      sc.ScType.EdgeDCommonVar,
-      [sc.ScType.NodeVar, "_result"],
-      sc.ScType.EdgeAccessVarPosPerm,
+      sc.ScType.VarCommonArc,
+      [sc.ScType.VarNode, "_result"],
+      sc.ScType.VarPermPosArc,
```

Comment:
have you replaced all deprecated usages manually or have you automated it in any way? other users of ts-sc-client and py-sc-client may want to stop using deprecated members as well

Reply:
I made this manually. If it is needed we should write script that will replace all deprecated types by new ones